backend/src/schemas/entities/settings.py

```
@@ -0,0 +1,48 @@
+
+    user_id: str = Field(..., description="ID of the user who owns these settings")
+    default_model: Optional[str] = Field(
+        default=None, description="User's default AI model identifier"
+    )
+    encrypted_keys: Optional[str] = Field(
+        default=None, description="Fernet-encrypted JSON blob of provider API keys"
+    )
+
+
+class UserSettingsResponse(BaseModel):
+    """API response model – never includes raw keys."""
```

Ruff (0.14.14)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)